src/pages/profile/messaging.ts

```diff
@@ -21,7 +21,7 @@ class ContactListComponent
 
   oninit() {
     this.contacts = ContactService.getContacts()
-    agent.onMessage("https://didcomm.org/basicmessage/2.0/message", this.onMessageReceived.bind(this))
+    agent.onMessage("messageReceived", this.onMessageReceived.bind(this))
```

Comment:
I think we should handle this a different way. I fear this will make registering message handlers a bit confusing if we're also registering general message receivers the same way we register specific ones. I'll have a think about whether there's another way to do this.

Reply:
Yeah, wasn't sure how to get all messages to show up in the message list :thinking: That was my goal anyways. Shows blue boxes for things like the user profile data and red boxes for "unknown message types", along with the standard white boxes for normal messages. I wasn't quite sure how to do a "catch all" to show a "we don't know what was sent to us" kind of message.
![Screenshot of multiple messages in the message window](https://media.discordapp.net/attachments/546454750258790407/1157168319745495080/2023-09-28-221305_864x775_scrot.png?ex=6517a07a&is=65164efa&hm=7163d6dfeb76a38eb422b23f453b630a4397092f5bf6d0b1b0c0b33fc4610654&=)

Reply:
What if we just add something like this to the `Agent` class?

```ts
  public onAnyMessage(callback: (message: AgentMessage) => void) {
    eventbus.on("messageReceived", callback)
  }
```

Reply:
I followed the advice in the next commit.